File: RUBY/actividades/actividad 31-05.rb

 puts "número de estudiantes"
 n=gets.chomp.to_1

    for x in 1..n
        
        puts "Nombre del estudiante"
        nb=gets.chomp

        puts"ingresé notas del 1 al 10"

        puts" nota 1"
        n1= gets.chomp.to_i

        puts" nota 2"
        n2= gets.chomp.to_i

        puts" nota 3"
        n3= gets.chomp.to_i

        Sum = n1 + n2 + n3

        Prom = sum / 3

        if prom > 6
         puts "el estudiante aprobó"
         else
         puts "el estudiante no aprobó"
        end

    end

File: DART/actividades/actividad 31-05.dart
import 'dart:io';

void main() { 
  print("número de estudiantes"); 
  int n = int.parse(stdin.readLineSync()!); 

  for(n = 1;  n <= n; n++) {
    print("nombre del estudiantes"); 
    String? nb = stdin.readLineSync();

    print("ingrese las notas entre 1 y 10");

    print("nota 1"); 
    int n1 = int.parse(stdin.readLineSync()!); 

    print("nota 2"); 
    int n2 = int.parse(stdin.readLineSync()!);

    print("nota 3"); 
    int n3 = int.parse(stdin.readLineSync()!); 

    int sum = (n1 + n2 + n3); 

    double prom = (sum / 3); 

      if ( prom > 6){
      print("el estudiante aprobo");
      } else { 
      print("El estudiante reprobo"); 

  }
 }
}